src/platform/site-wide/cta-widget/index.js

```diff
@@ -436,6 +467,10 @@ export class CallToActionWidget extends React.Component {
     }
   };
 
+  goToProfile = () => {
+    window.location.pathname = '/profile';
+  };
```

Comment:
I think we can use `this._toolUrl` here instead of `/profile`?

Reply:
Now using goToTool() method above -- deleted my goToProfile().